.palantir/revapi.yml

```diff
@@ -1171,6 +1171,28 @@ acceptedBreaks:
         \ java.util.function.Function<T, java.lang.Long>, org.apache.iceberg.io.CloseableIterable<java.lang.Long>,\
         \ java.util.function.Consumer<T>)"
       justification: "Removing deprecated code"
+    org.apache.iceberg:iceberg-parquet:
+    - code: "java.method.abstractMethodAdded"
+      new: "method org.apache.iceberg.parquet.ParquetValueReaders.PrimitiveReader<?>\
+        \ org.apache.iceberg.data.parquet.BaseParquetReaders<T>::fixedReader(org.apache.parquet.column.ColumnDescriptor)"
+      justification: "{Refactor Parquet reader and writer}"
+    - code: "java.method.abstractMethodAdded"
+      new: "method org.apache.iceberg.parquet.ParquetValueReaders.PrimitiveReader<?>\
+        \ org.apache.iceberg.data.parquet.BaseParquetReaders<T>::int96Reader(org.apache.parquet.column.ColumnDescriptor)"
+      justification: "{Refactor Parquet reader and writer}"
+    - code: "java.method.abstractMethodAdded"
+      new: "method org.apache.iceberg.parquet.ParquetValueWriters.PrimitiveWriter<?>\
+        \ org.apache.iceberg.data.parquet.BaseParquetWriter<T>::fixedWriter(org.apache.parquet.column.ColumnDescriptor)"
+      justification: "{Refactor Parquet reader and writer}"
+    - code: "java.method.abstractMethodAdded"
+      new: "method org.apache.parquet.schema.LogicalTypeAnnotation.LogicalTypeAnnotationVisitor<org.apache.iceberg.parquet.ParquetValueReader<?>>\
+        \ org.apache.iceberg.data.parquet.BaseParquetReaders<T>::logicalTypeReaderVisitor(org.apache.parquet.column.ColumnDescriptor,\
+        \ org.apache.iceberg.types.Type.PrimitiveType, org.apache.parquet.schema.PrimitiveType)"
+      justification: "{Refactor Parquet reader and writer}"
+    - code: "java.method.abstractMethodAdded"
```

Comment:
This PR should not introduce revapi failures. Instead, the new methods should have default implementations that match the previous behavior (returning the generic representations).

Reply:
New methods are abstract and abstract method cannot have default implementation. So, I think we have to handle revapi failures.

Reply:
Oh, I think what you mean is don't add it as abstract. Add it as methods with default implementation. I got it. I will update it today.